ext/AvizExt/viz/sensitivity.jl

```diff
@@ -155,15 +155,14 @@ function ADRIA.viz.tsa(
 end
 
 """
-    ADRIA.viz.rsa(rs::ResultSet, si::NamedDimsArray, factors::Vector{String}; opts::Dict=Dict(), fig_opts::Dict=Dict(), axis_opts::Dict=Dict())
-    ADRIA.viz.rsa!(f::Union{GridLayout,GridPosition}, rs::ResultSet, si::NamedDimsArray, factors::Vector{String}; opts, axis_opts)
+    ADRIA.viz.rsa(rs::ResultSet, si::NamedDimsArray, factors::Vector{String}; fig_opts::Dict=Dict(), axis_opts::Dict=Dict())
+    ADRIA.viz.rsa!(f::Union{GridLayout,GridPosition}, rs::ResultSet, si::NamedDimsArray, factors::Vector{String}; axis_opts::Dict=Dict())
 
 Plot regional sensitivities of up to 30 factors.
 
 # Arguments
 - `rs` : ResultSet
 - `si` : Results from ADRIA regional sensitivity analysis
-- `opts` : Additional figure customization options
 - `fig_opts` : Additional options to pass to adjust Figure creation
   See: https://docs.makie.org/v0.19/api/index.html#Figure
 - `axis_opts` : Additional options to pass to adjust Axis attributes
```

Comment:
The `opts` argument - at least for `rsa()` - is unused but purposefully put in to maintain a consistent API as most other plotting methods expects `opts`.